src/build.js

```diff
@@ -401,32 +397,44 @@ class Build
     /**
      * ## getDescription
      *
-     * gets the first text description in the content and returns that
+     * gets description from the content and returns that. Either returns full text or only from
+     * the first block
      *
-     * @param {Array} arr array filled w/ content
+     * @param {Array} storyTextArr array filled w/ content
+     * @param {Boolean} useFullText when false, use only the first text content block
      * @param {Number} i recursive index
      *
-     * @return {Mixed} text content or false _String or Boolean_
+     * @return {String} text content. Might be empty if no text.
      */
-    getDescription( arr, i = 0 )
+    getDescription( storyTextArr, useFullText = true, i = 0 )
```

Comment:
don't include the type in the variable name. Instead, call it textSections or something the like